### PR TITLE
Travis Build

### PR DESCRIPTION
**Pull request by IgnatiusFlo**
Hello,

When I added py3.6 it wouldn't build so I made a branch with the commit that worked. It made Travis have an issue with unacceptable characters in paths. On Pypi it says Py3.6 is supported and I don't know why this is happening. 

There is a mention in setup.py about versions. Maybe.... :bug:

### CODE REVIEW

**Comment by lgiordani** [comment]
Hi, this PR is good, can you just remove the changes in the requirements? They are part of another PR. To do it you can just `git checkout origin/master filename` on those files and commit. Let me know if you have issues.

May you send another PR with the 3.6 change, even if it doesn't pass the tests? We can discuss the code there. Thanks

File: {{cookiecutter.project_slug}}/requirements/dev.txt [hunk 1]
```diff
@@ -1,10 +1,14 @@
 -r test.txt
```
Comment (lgiordani) [inline {{cookiecutter.project_slug}}/requirements/dev.txt:1]: Please get rid of these changes, we can put them in the other PR :beach_umbrella:

File: {{cookiecutter.project_slug}}/requirements/test.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 -r prod.txt
```
Comment (lgiordani) [inline {{cookiecutter.project_slug}}/requirements/test.txt:1]: These as well :penguin:

**Review by lgiordani — CHANGES_REQUESTED** [review]

**Comment by IgnatiusFlo** [comment]
I was going to add 3.6 to the setup.py classifiers and then add it back into the tox and travis files because I think that will work, but I will do as you ask.  

I didn't mean the requirements.txt changes to be in this PR. Thanks for your patience while I figure this out. I will get back to you at the weekend. :sunflower:

**Comment by lgiordani** [comment]
Kate, no worries at all. Thank you for helping!

**Comment by lgiordani** [comment]
Hi! What happened with this PR? Are you still working on it?

**Comment by lgiordani** [comment]
Hi! Thanks for the PR, I imported your code in another branch and created #4 You were missing a couple of changes, but apparently you don't have time to work on them so I did it. Thanks a lot for the contribution! :+1:

**Pull request closed without merging**